### DISCUSSION TITLE
[Analytics] Self-hosted analytics software

### DISCUSSION THREAD

**Discussion by mkubdev** [StreetCommunityProgrammer/metaphore#27] [Ideas]
We need to collect analytics.

Define a self-hosted solution to collect data analytics (from @darkterminal) :
- [Snowplow](https://docs.snowplow.io/docs/getting-started-on-snowplow-open-source/)
- [Ackee](https://docs.ackee.electerious.com/#/)

### Hosting

Where do we host it? Do we use AWS/GCP Freeplan? Or our own servers?

**Comment by darkterminal** [comment]
We use Ackee and it's now live how about it?

**Reply by mkubdev** [reply]
It's looking so great! I think the data collected are enough at this stage.

Thanks for make it live, I'm ok to choose Ackee!

**Reply by darkterminal** [reply]
OK, deal!!!!!
